static/styles/landing-page.scss

```diff
@@ -2106,7 +2106,7 @@ nav ul li.active::after {
 }
 
 .portico-landing.apps .other-apps .apps .icon:hover {
-    color: #777;
+    color: hsl(0, 0%, 47%);
```

Comment:
Previously it was changed to `color: hsl(0, 0%, 46%);`  in [commit](https://github.com/zulip/zulip/commit/7e869cf5b20bb7c41a7ececb924ee4392bffd859#diff-a22e629c023faad5c3e8c2554d0e2c74R404)

We still have a few occurrences of this one in our code base. Just grep for `color: hsl(0, 0%, 46%);`